test/test.highlevelConsumer.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+var util = require('util');
+var EventEmitter = require('events');
+var sinon = require('sinon');
+
+var libPath = process.env['KAFKA_COV'] ? '../lib-cov/' : '../lib/';
+
+// var Client = require(libPath + 'client');
```

Comment:
I tried to use Client initially. But it was a pain so eventually mocked it. Will delete the comment.
Regarding KAFKA_COV that is how libPath is being set in test.consumer.js. So I thought it was being set somewhere. Will remove it if it is not needed.